app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+  validates :title, presence: true, uniqueness: { scope: :category }
+
+  def self.books
```

Comment:
Consider whether you could DRY this up with a single method such as `get_by_category(category)`